### PR TITLE
Include configured option in toolchains report

### PR DESCRIPTION
**Pull request by bmuskalla**
This helps us and users to quickly see how Gradle is configured with regards to toolchains.

### CODE REVIEW

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -181,6 +181,10 @@ In order to see which toolchains got detected and their corresponding metadata,
 
 Output of `gradle -q javaToolchains`:
 ```
+ + Options
+     | Auto-detection enabled?  Yes
```
Comment (melix) [inline subprojects/docs/src/docs/release/notes.md:185]: Nice!
Alternative:
| Auto-detection:  Enabled

Comment (ljacomet) [reply subprojects/docs/src/docs/release/notes.md:185]: 👍 on this proposal

**Review by ljacomet — CHANGES_REQUESTED** [review]
I prefer the addition with the proposal from Cédric. All good otherwise and it is a good idea to have that information.

With the discussions around filtering invalid installations, maybe those should also be added to this report?

File: subprojects/docs/src/docs/release/notes.md (hunk 1 above)
Comment (bmuskalla) [reply subprojects/docs/src/docs/release/notes.md:185]: Sounds good!

**Comment by bmuskalla** [comment]
@bot-gradle test this

**Comment by bmuskalla** [comment]
@ljacomet Broken installations are part of the reports, see [tests like this](https://github.com/gradle/gradle/blob/master/subprojects/platform-jvm/src/test/groovy/org/gradle/jvm/toolchain/internal/ShowToolchainsTaskTest.groovy#L126-L130).

**Comment by ljacomet** [comment]
I missed that, sorry. Great that it was in already!

**Pull request merged**